features/in_onderzoek.feature

```diff
@@ -597,6 +597,173 @@ Functionaliteit: in onderzoek
+      | naam                          | waarde   |
+      | aanduidingGegevensInOnderzoek | 081100   |
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde       |
+      | verblijfadres.huisnummer                                   | 70           |
+      | verblijfadres.straat                                       | spui         |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      | verblijfadres.inOnderzoek.aanduidingBijHuisnummer          | true         |
+      | verblijfadres.inOnderzoek.huisletter                       | true         |
+      | verblijfadres.inOnderzoek.huisnummer                       | true         |
+      | verblijfadres.inOnderzoek.huisnummertoevoeging             | true         |
+      | verblijfadres.inOnderzoek.straat                           | true         |
+      | verblijfadres.inOnderzoek.postcode                         | true         |
+      | verblijfadres.inOnderzoek.korteNaam                        | true         |
+      | verblijfadres.inOnderzoek.woonplaats                       | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum                     | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type                      | Datum        |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat               | 7 maart 2022 |
+      | inOnderzoek.type                                           | true         |
+      | inOnderzoek.nummeraanduidingIdentificatie                  | true         |
+      | inOnderzoek.adresseerbaarObjectIdentificatie               | true         |
+
+    Scenario: een element van de groep adres van de categorie verblijfplaats is in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      En de persoon heeft GEEN 'persoonInOnderzoek' gegevens
+      En persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfplaats' gegevens
+      | naam       | waarde |
+      | straat     | spui   |
+      | huisnummer | 70     |
+      En persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfplaats.inOnderzoek' gegevens
```

Comment:
deze regel moet er tussenuit. inOnderzoek zit ook in de tabel/categorie verblijfplaats